src/App.js

```diff
@@ -0,0 +1,464 @@
+                	<input type="submit" value="-"/>
+				Fly: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Spellcraft: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Stealth: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Survival: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Swim: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Use Magic Device: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Ride: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Nature: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Diplomacy: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Engineering: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Preform: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Disable Device: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Geography: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Escape Artist: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Slight of Hand: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+			<p>Languages:</p>
+			<p>Languages that are avilable</p>
+		</div>
+		<div>
+			<p>Feats:</p>
+			<p>Feats that are avilable</p>
+		</div>
+		<div>
+			<p>Special Abilities:</p>
+			<p>Special Abilities that are avilable</p>
+		</div>
+            </TabPanel>
+            <TabPanel>
+		<div>
+			<p>Cantrips:</p>
+			<p>Cantrips that are avilable</p>
+		</div>
+		<div>
+			<p>Prepared Spells:</p>
+			<p>Prepared Spells that are avilable</p>
+		</div>
+		<div>
+			<p>Active Spell Effects Abilities:</p>
+			<p>Active Spell Effects that are avilable</p>
+		</div>
+		<h2>Spell Slots</h2>
+		<div>
+			Level	Max	Current
+		</div>
+		<div>
+			Level 1:
+		</div>
+		<div>
+			Level 2:
+		</div>
+		<div>
+			Level 3:
+		</div>
+		<div>
+			Level 4:
+		</div>
+		<div>
+			Level 5:
+		</div>
+		<div>
+			Level 6:
+		</div>
+		<div>
+			Level 7:
+		</div>
+		<div>
+			Level 8:
+		</div>
+		<div>
+			Level 9:
+		</div>
```

Comment:
Good candidate for using a table. There are ways to make tables responsive to changes in screen size, too. Might also make this a React element if tables are useful elsewhere.